packages/sources/src/components/tree-view/tree-view.models.ts

```diff
@@ -0,0 +1,21 @@
+export interface TreeViewItem<T> {
```

Comment:
Dit is nog mijn grootste obstakel. Moeten we nog iets met `<T>` doen? Morgen maar even over bellen.. Misschien is `<T = any>` wel genoeg?

Brainfart: Kunnen we de API niet een met een identity function verrijken? De afnemer moet dan een functie toevoegen die wij vanuit de JSX aanroepen: `<li key={this.identifyItem(item)}`. Zolang de returntype maar 'n primitive is maakt het ons niet meer uit en hebben we het hele `reference: T` gebeuren opgeruimd